README.md

```diff
@@ -108,11 +208,8 @@ How many levels are within the following path?
 
 `/Users/kellyegreene/Development/code/flatiron-school/mixtape-app`
 
-- [The One True Path](http://blog.seldomatt.com/blog/2012/10/08/bash-and-the-one-true-path/)
-- [More on paths - Wikipedia](http://en.wikipedia.org/wiki/Path_\(computing\))
-
-If you said 6 levels to the question above you are right! Knowing where you are in your terminal -
-what directory you are working in - is very important.
+If you said 6 levels to the question above you are right! Knowing where you
+are in your terminal is very important.
 
 ### Use `ls` to List Files in Shell
 
```

Comment:
Set some context here:

"So far we've been finding out where we are in the filesystem "tree," how about we find out what's in these directories (besides other directories)?"